### ISSUE TITLE
PCD with global coordinates?

### ISSUE THREAD

**Issue by antoniomontana1992** [TixiaoShan/LIO-SAM#524]
Is it possible to create .pcd files with global coordinates?

**Comment by stah-ba** [comment]
@antoniomontana1992 did you solve the issue? Or how did you continue?

**Comment by lukasj-imar** [comment]
There is a field in the pcl object, which is called sensor_origin. You can use this field to define your offset in a geodetic reference frame.

**Comment by antoniomontana1992** [comment]
I dont know if this is the right way, but what i did is to keep the first fixed reading from my RTK sensor, and then start liosam. While the map is finished, i make the conversion with pdal based on the first reading of my location.

**Comment by stah-ba** [comment]
Alright, thanks for the quick answers! @lukasj-imar so you also use only the first GPS measurement as a offset as sensor origin for the mapped cloud?

**Comment by lukasj-imar** [comment]
Yes, i take a geocentric coordinate from GPS as  sensor origin. Each point of the cloud is represented in an ENU frame which has his origin in this sensor origin.
